plaidml2/op/op.h

```diff
@@ -107,6 +107,47 @@ inline edsl::Tensor prod(const edsl::Tensor& I, const edsl::Value& axes = edsl::
   return details::op("prod", args).as_tensor();
 }
 
+class Relu {
+ protected:
+  edsl::Tensor I_;
+  edsl::Tensor alpha_ = edsl::None().as_tensor();
+  edsl::Tensor max_value_ = edsl::None().as_tensor();
+  float threshold_ = 0.0;
+
+ public:
+  explicit Relu(const edsl::Tensor& I) { I_ = I; }
+  void setAlpha(const edsl::Tensor& alpha) { alpha_ = alpha; }
+  void setMaxValue(const edsl::Tensor& max_value) { max_value_ = max_value; }
+  void setThreshold(float threshold) { threshold_ = threshold; }
```

Comment:
let's leave the `set` prefix off, since this a builder API, seems natural that calling a method on a builder means you're setting a property of the builder.